applicable_pose_generator/src/pose_reachability_filter.cpp

```diff
@@ -0,0 +1,68 @@
+#include <tesseract_ros/kdl/kdl_chain_kin.h>
+#include <urdf_model/model.h>
+
+#include <kdl/chainiksolverpos_lma.hpp>
+
+#include "ros/ros.h"
+
+#include "applicable_pose_generator/pose_reachability_filter.h"
+#include <boost/shared_ptr.hpp>
+
+#include <kdl_parser/kdl_parser.hpp>
+#include <eigen_conversions/eigen_kdl.h>
+
+namespace create_chain_take_pose_inverse_kinamatics
+{
+
+
+bool chain_creation::chain_Parse(Eigen::Affine3d ei_transform_to_check)
+{
+  urdf::Model Mymodel;
+  ros::NodeHandle nh("~");
```

Comment:
normally, nh is reserved for a non-private node handle name. Its really just a convention, but change nh to pnh or private_nh or some such indication that this is a private node handle